src/NzbDrone.Core/Indexers/Definitions/Cardigann/Cardigann.cs

```diff
@@ -127,11 +127,11 @@ private IndexerDefinition GetDefinition(CardigannMetaDefinition definition)
                 Implementation = GetType().Name,
                 IndexerUrls = definition.Links.ToArray(),
                 Settings = new CardigannSettings { DefinitionFile = definition.File },
-                Protocol = DownloadProtocol.Torrent,
+                Protocol = definition.Protocoltype == "usenet" ? DownloadProtocol.Usenet : DownloadProtocol.Torrent,
```

Comment:
Need to adjust the Download Call in Cardigann to be conditional based on the protocol